1_assignment/alphbet_frequency.js

```diff
@@ -1,13 +1,18 @@
 
-string = "u@#d$fkj$#!kds"
 
-function calculateFrequency(string) {
+//Sample String .
```

Comment:
Consistency in coding style is really important. The comment here should have a space after '//' which is present correctly on line 8 below.
Also no space before '.' and ',' only after. Remember this in all written communication as well :)